modules/db/src/main/scala/gem/dao/StaticConfigDao.scala

```diff
@@ -134,21 +134,28 @@ object StaticConfigDao {
 
       // We need to define this explicitly because we're ignoring the nod and
       // shuffle and custom ROIs.
-      implicit val GmosCommonStaticComposite: Composite[GmosCommonStaticConfig] =
-        Composite[(GmosDetector, MosPreImaging)].imap(
-          (t: (GmosDetector, MosPreImaging)) => GmosCommonStaticConfig(t._1, t._2, None, Set.empty))(
+      implicit val GmosCommonStaticRead: Read[GmosCommonStaticConfig] =
+        Read[(GmosDetector, MosPreImaging)].map {
+          (t: (GmosDetector, MosPreImaging)) => GmosCommonStaticConfig(t._1, t._2, None, Set.empty)
+        }
+
+      implicit val GmosCommonStaticWrite: Write[GmosCommonStaticConfig] =
+        Write[(GmosDetector, MosPreImaging)].contramap(
```

Comment:
So `Composite` becomes `Read` and `Write`?

Reply:
Yes, `Meta` becomes `Get` + `Put` and `Composite` becomes `Read` + `Write`. There's a description a few paragraphs down [here](https://tpolecat.github.io/doobie/changelog.html).